clients/cmd/promtail/promtail-cloudflare.yaml

```diff
@@ -0,0 +1,17 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://localhost:3100/loki/api/v1/push
+
+scrape_configs:
+- job_name: kafka
```

Comment:
Out of curiosity why is this named `kafka`?

Reply:
🤦